### PR TITLE
Dockerfile update

### PR DESCRIPTION
**Pull request by rsevilla87**
As soon this one gets merged we'd have to enable the CI job.
Depends on https://github.com/cloud-bulldozer/bohica/pull/16

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,27 +1,13 @@
...
-RUN yum install -y ansible dmidecode which python-pip pciutils
-RUN pip3 install --upgrade pip
-RUN pip3 install elasticsearch-dsl
-RUN pip3 install openshift
-RUN pip3 install kubernetes
-RUN pip3 install redis
-RUN pip3 install -e scribe/
-RUN mv ansible.cfg /etc/ansible/ansible.cfg
-RUN mkdir -p /tmp
-RUN sed -i '/become: true/d' stockpile.yml
```
Comment (dry923) [inline Dockerfile:24]: @rsevilla87 we need this line. If we leave it in it will try to sudo to root and fail since sudo isnt there/confi'd

Comment (rsevilla87) [reply Dockerfile:24]: fixed!

**Review by jtaleric — APPROVED** [review]
LGTM

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**